runtime-decompiler/src/main/java/org/jrd/frontend/PluginMangerFrame/FileSelectorArrayRow.java

```diff
@@ -68,6 +65,13 @@
         this.add(browseButton, gbc);
     }
 
+    public static File fallback(File currentDir){
+        while(!currentDir.exists()){
+            currentDir = currentDir.getParentFile();
```

Comment:
in sudden moment, the parent can be null.  I would recommend to check that, and if dir.parent is null, return dir, rather then NPE

Reply:
What's this sudden moment?